### ISSUE TITLE
Get rid of admin tab?

### ISSUE THREAD

**Issue by ashfame** [WordPress/try-wordpress#78]
Right now, our preview pane has two tab views: "Preview" which renders the frontend of the site and "Admin" which renders the WordPress admin of the site `/wp-admin`.

I think we should get rid of admin tab in preview pane, since we make the user logged in and they always have the admin bar to go to admin or come back to frontend side.

WHY?

It gets weird for the user to reach admin under "preview" tab view. And on frontend side under "admin" tab view. I think the goal of showing them where admin is, which I believe was the goal of having tabbed views, is easily achievable by a button in UI pane, which redirects the playground to navigate to admin area.

**Comment by psrpinto** [comment]
The reasoning for having the admin on another tab would be so that the user can have both open simultaneously, otherwise they need to keep going back and forth.

I'm fine with removing it if we think this makes things confusing for the user.

**Comment by ashfame** [comment]
Ok, let's keep it until the UI has evolved a bit and then revisit?

I proposed it since I ran into the confusion issue myself and sometimes admin tab doesn't load anything for me (however I haven't been able to reliably reproduce it). Hence the proposal to kill 2 birds with 1 stone :)

Perhaps a button that takes them to admin and back (toggle) could also work but I think the need of remembering what url were they on (atleast on frontend) sounds complicated than tabbed view.
